### PR TITLE
ad new test ( addRangeWithGrow )

### PR DESCRIPTION
**Pull request by Kameliya-M**

### CODE REVIEW

File: Collection.UnitTests/CollectionTests.cs [hunk 1]
```diff
@@ -64,6 +64,23 @@ public void Test_GetByInvalidIndex()
             Assert.That(() => { var item = coll[2]; }, Throws.InstanceOf<ArgumentOutOfRangeException>());
 
             
+        }
+        [Test]
+        public void Test_Collection_AddRangeWithGrow()
+        {
+            var nums = new Collection<int>();
+            int oldCapacity = nums.Capacity;
+            var newNums = Enumerable.Range(1000, 2000).ToArray();
```
Comment (Tuscann) [inline Collection.UnitTests/CollectionTests.cs:73]: Add  // Arrange // Act  // Assert

File: Collection.UnitTests/CollectionTests.cs [hunk 2]
```diff
@@ -64,6 +64,23 @@ public void Test_GetByInvalidIndex()
...
+
+            nums.AddRange(newNums);
+
+            string expectedNums = "[" + string.Join(",", newNums) + "]";
+            Assert.That(nums.ToString(), Is.EqualTo(expectedNums));
+            Assert.That(nums.Capacity, Is.GreaterThanOrEqualTo(oldCapacity));
```
Comment (Tuscann) [inline Collection.UnitTests/CollectionTests.cs]: One test one assert!

**Review by Tuscann — CHANGES_REQUESTED** [review]
I can not write comment on previous tests.

File: Collection.UnitTests/CollectionTests.cs (hunk 2 above)
Comment (Kameliya-M) [reply Collection.UnitTests/CollectionTests.cs]: Perfect, thank you!

**Review by Tuscann — APPROVED** [review]

**Pull request merged**